_spinnaker/terraform_integration.md

```diff
@@ -156,6 +156,33 @@ When creating a Terraform Integration stage, pipeline creators select a specific
 
 Note that all Terraform stages within a Pipeline that affect state must use the same Terraform version.
 
+### Configuring a Profile
+
+Configure profiles that users can select when creating a Terraform Integration stage. A profile gives users the ability to reference certain kinds of external sources, such as a private remote repository, when creating pipelines.
+
+For example, if your Terraform scripts rely on modules stored in a private remote Git repository, add your `SSH` key to a profile. Then, a user can select that profile when creating a Terraform Integration stage. When a pipeline runs, the Terraform Integration automatically gains access to fetch what it needs from the private repo.
+
+To add profiles that a user can select from, perform the following steps:
+
+1. In the `.hal/default/profiles` directory, create or edit `terraformer-local.yml`.
+2. Add the values for the profile(s) you want to add under an `environments` section. The following example adds a profile named `pixel-git` for an SSH key secured in Vault and used to access a Git repo: 
+    
+    ```
+    # Profile name displayed in Deck
```

Comment:
@aelath, can you check that this YAML is correct? Mark said it might have changed but ran out of time ;)

Reply:
Yep, it has changed. Let me suggest something.

Reply:
So, multiline suggestions are hard. This is what you want:
```
    - name: pixel-git
        variables:
        - kind: git-ssh
          options: 
            sshPrivateKey: encrypted:vault!e:<secret engine>!p:<path to secret>!k:<key>!b:<is base64 encoded?> 
```

Reply:
You can also use a private key file instead of the contents. Instead of `sshPrivateKey` it would be `sshPrivateKeyFilePath`. This can also be encrypted using a secrets provider, indicated by `encryptedFile`.

Reply:
But only one option can be configured at a time for a single profile. `sshPrivateKey` OR `sshPrivateKeyFilePath`.

Reply:
Gracias